src/components/Alert/index.tsx

```diff
@@ -19,23 +18,41 @@ import {
 } from '@ticketswap/comets'
 import { H4 } from '../Heading'
 
-const textColor = props => css`
-  color: ${props.variant === 'success'
+export type AlertAction = {
+  label: string
+  onClick: (event: React.MouseEvent<HTMLElement>) => void
+}
+
+export enum AlertVariant {
+  info = 'info',
+  success = 'success',
+  error = 'error',
+  warning = 'warning',
+}
```

Comment:
There’s a [case against](https://blog.logrocket.com/why-typescript-enums-suck/) using enums in TypeScript. Might be clearer and less error-prone to use string union types.
```suggestion
export type AlertVariant = 'info' | 'succes' | 'error' | 'warning'
```

Reply:
Thanks Philipp! Let's discuss tomorrow @RobVermeer @Roohn ;)